lib/arqo.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'arqo/query'
 require 'arqo/version'
+require 'arqo/railtie' if defined?(Rails)
```

Comment:
I think Rails will ~almost~ always be defined since this gem depends on ActiveRecord

Reply:
Ok. I'll remove it